app/components/channel_drawer/channel_drawer.js

```diff
@@ -34,9 +48,12 @@ export default class ChannelDrawer extends PureComponent {
     };
 
     state = {
-        openDrawer: false
+        openDrawer: false,
+        openDrawerOffset: 40
     };
 
+    swiperIndex = 1;
```

Comment:
is not needed in the state as this is only to identify which index is being shown in the Pager or the Swiper to call the reset of the index, which is totally different than the prop received by those components